nikola/__main__.py

```diff
@@ -65,8 +68,11 @@
 _RETURN_DOITNIKOLA = False
 
 
-def main(args=None):
+
+def main(options=None):
     """Run Nikola."""
+    options, args = cli.parse_args(options) if isinstance(options, (list, tuple, None.__class__)) else options
```

Comment:
This is an ugly hack, please think of something else, perhaps you could just keep the old API?